script/lint

```diff
@@ -17,5 +17,6 @@ fi
 
 SOURCES="*.py octodns/*.py octodns/*/*.py tests/*.py"
 
-pycodestyle --ignore=E221,E241,E251,E722,W504 $SOURCES
+pycodestyle --ignore=E117,E221,E241,E251,E722,W504 $SOURCES
```

Comment:
What's the reason for adding `E117`. That check get updated to be more strict/different?

Reply:
E117 over-indented seems to disagree with the code formatted that this project uses.

Reply:
Hrm. Guess that's something that's changed between 2.4.0 and now.

Reply:
The new warnings were fixed in master.